test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlServerSchemaUpgradeTests.cs

```diff
@@ -50,16 +50,25 @@ public async Task GivenTwoSchemaInitializationMethods_WhenCreatingTwoDatabases_B
             }
         }
 
-        [Fact]
-        public async Task GivenASchemaVersion_WhenApplyingDiffTwice_ShouldSucceed()
+        [Theory]
+        [InlineData((int)SchemaVersion.V8)]
```

Comment:
+1, I am wondering if we would like to validate - idempotent scripts from the version 7(when we started making our scripts idempotent OR last two versions OR SchemaVersion.Max.

My thoughts are since we don't touch/update the existing scripts(I believe) so we should be good just testing the max version. And at the same time, I am not sure how much extra time it would take to run for more than one schema version but validating most recent couple of versions should be fine as well.

Reply:
This test actually doesn't seem to do what I thought it was doing :). I have sent out a different PR #1838 to update the test.